### PR TITLE
docs/blog: Add blogpost for Unikraft Hacktoberfest '22

### PR DESCRIPTION
**Pull request by RaduNichita**
Signed-off-by: Radu Nichita <radunichita99@gmail.com>

### CODE REVIEW

File: content/en/blog/2022-10-08-unikraft-hacktoberfest.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
++++
+title = "Unikraft Hacktoberfest '22"
+date = "2022-10-08T09:00:00+01:00"
+author = "Radu Nichita"
+tags = ["hacktoberfest"]
++++
+
+### A new beginning
```
Comment (razvand) [inline content/en/blog/2022-10-08-unikraft-hacktoberfest.md:8]:
```suggestion
### And Another One
```

File: content/en/blog/2022-10-08-unikraft-hacktoberfest.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+Last weekend, Unikraft hosted a community event for the `Hacktoberfest` series.
```
Comment (razvand) [inline content/en/blog/2022-10-08-unikraft-hacktoberfest.md:10]:
```suggestion
Last weekend, we hosted yet another Unikraft hackathon, part of [Hacktoberfest 2022](https://hacktoberfest.com/).
```

File: content/en/blog/2022-10-08-unikraft-hacktoberfest.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+The event was held in a hybrid way: the newcomers and some familiar faces met physically at `University POLITEHNICA of Bucharest`, whereas the other folks joined remotely on the `monkey-business` channel.
```
Comment (razvand) [inline content/en/blog/2022-10-08-unikraft-hacktoberfest.md:11]:
```suggestion
This was a hybrid event: the newcomers and some familiar faces met physically at University POLITEHNICA of Bucharest, whereas the other folks joined remotely on [the Unikraft Discord server](http://bit.ly/UnikraftDiscord).
```

File: content/en/blog/2022-10-08-unikraft-hacktoberfest.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+### First steps
```
Comment (razvand) [inline content/en/blog/2022-10-08-unikraft-hacktoberfest.md:13]:
```suggestion
### First Steps
```

File: content/en/blog/2022-10-08-unikraft-hacktoberfest.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+Firstly, there was a presentation given by [Stefan Jumărea](https://github.com/StefanJum), our community lead for documentation.
+It was an introduction to unikernels and Unikraft and a brief presentation of hackathon challenges.
+
+### Hardwork always pays off
```
Comment (razvand) [inline content/en/blog/2022-10-08-unikraft-hacktoberfest.md:18]:
```suggestion
### Hard work Always Pays Off
```

File: content/en/blog/2022-10-08-unikraft-hacktoberfest.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+Afterwards, the participants dove deep into the chosen tasks.
```
Comment (razvand) [inline content/en/blog/2022-10-08-unikraft-hacktoberfest.md:20]:
```suggestion
Afterwards, the participants dove deep into the chosen tasks, from the [`Hackathons` project on GitHub](https://github.com/orgs/unikraft/projects/29).
```

File: content/en/blog/2022-10-08-unikraft-hacktoberfest.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+Most of them started by running the ` application on top of Unikraft.
```
Comment (razvand) [inline content/en/blog/2022-10-08-unikraft-hacktoberfest.md:21]:
```suggestion
Most of them started by running adding new [static PIE Linux executables](https://github.com/unikraft/static-pie-apps) to be run on top of Unikraft in binary compatibility mode using the [ELF Loader](https://github.com/unikraft/run-app-elfloader).
```

File: content/en/blog/2022-10-08-unikraft-hacktoberfest.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+Then, they followed some tutorials from the `USOC '22` to get accustomed to the core components of Unikraft.
+
+After a few hours of intense hacking, results were on the table: 3 applications were ported as `static-build-pie` for Unikraft!!
```
Comment (razvand) [inline content/en/blog/2022-10-08-unikraft-hacktoberfest.md:24]:
```suggestion
After a few hours of intense hacking, results were on the table: 3 applications were ported as static PIE apps for Unikraft!
And participants are working forward to port applications and submit PRs.
Another PRs was submitted the next day.
```

**Review by razvand — CHANGES_REQUESTED** [review]
Thanks, @RaduNichita , see my comments.

**Review by mandrei12 — CHANGES_REQUESTED** [review]
I think the idea is there but we can work on the language and present the event in a more suitable and welcoming manner.

File: content/en/blog/2022-10-08-unikraft-hacktoberfest.md [hunk 9]
```diff
@@ -5,23 +5,26 @@ author = "Radu Nichita"
 tags = ["hacktoberfest"]
 +++
 
-### A new beginning
+### And Another One
 
-Last weekend, Unikraft hosted a community event for the `Hacktoberfest` series.
-The event was held in a hybrid way: the newcomers and some familiar faces met physically at `University POLITEHNICA of Bucharest`, whereas the other folks joined remotely on the `monkey-business` channel.
+As tradition dictates if it's Saturday then it is definitely another hackathon we are proud to host at Unikraft. Even so, the last one was a bit special. How you ask? Well, as we previously announced, we have been officially accepted to be part of the [Hacktoberfest 2022](https://hacktoberfest.com/).
```
Comment (RaduNichita) [inline content/en/blog/2022-10-08-unikraft-hacktoberfest.md:10]:
```suggestion
As tradition dictates, if it's Saturday then it is definitely another hackathon we are proud to host at Unikraft. Even so, the last one was a bit special. How you ask? Well, as we previously announced, we have been officially accepted to be part of the [Hacktoberfest 2022](https://hacktoberfest.com/).
```

File: content/en/blog/2022-10-08-unikraft-hacktoberfest.md [hunk 10]
```diff
@@ -5,23 +5,26 @@ author = "Radu Nichita"
...
 
-### First steps
+Thus `UNIKRAFT HACKTOBERFEST '22` was born in the shape of a hybrid event. Here both familiar faces and newcomers alike came together physically at University POLITEHNICA of Bucharest or joined remotely on [the Unikraft Discord server](http://bit.ly/UnikraftDiscord) in order to test their knowledge in unikernels.
 
-Firstly, there was a presentation given by [Stefan Jumărea](https://github.com/StefanJum), our community lead for documentation.
-It was an introduction to unikernels and Unikraft and a brief presentation of hackathon challenges.
+### First Steps
 
-### Hardwork always pays off
+In order to break the ice, our documentation community lead, [Stefan Jumărea](https://github.com/StefanJum), gave a introductory presentation. This aimed to present both unikernels and Unikraft to the participants, introducing it to the newcomers and refreshing it for the veterans. Afterwards, a short discussion about the challenges raised our participants spirits. Onto krafting young padawans!
```
Comment (RaduNichita) [inline content/en/blog/2022-10-08-unikraft-hacktoberfest.md:16]:
```suggestion
In order to break the ice, our documentation community lead, [Stefan Jumărea](https://github.com/StefanJum), gave an introductory presentation. This aimed to present both unikernels and Unikraft to the participants, introducing it to the newcomers and refreshing it for the veterans. Afterward, a short discussion about the challenges raised our participant's spirits. Onto krafting young padawans!
```

File: content/en/blog/2022-10-08-unikraft-hacktoberfest.md [hunk 11]
```diff
@@ -5,23 +5,26 @@ author = "Radu Nichita"
...
-Most of them started by running the ` application on top of Unikraft.
-Then, they followed some tutorials from the `USOC '22` to get accustomed to the core components of Unikraft.
+### Hardwork Always Pays Off
+
+It was finally time for the moment we have been waiting for, the deep dive into the tasks. In order to warm up and close the gap between traditional applications and Unikraft, most of the participants chose to run static PIE Linux executables](https://github.com/unikraft/static-pie-apps) on top of Unikraft in binary compatibility mode using the [ELF Loader](https://github.com/unikraft/run-app-elfloader).
+The more experienced ones adventured even deeper and added new ones.
+
+In their aid [`USOC '22`](https://unikraft.org/community/hackathons/usoc22/) tutorials also played a great role, allowing participants to have a better understanding of the core components of Unikraft.
+
+While begginings can sometimes be hard and unrewarding, this was not the case for our contestans. After a few hours the results were on the table: 3 applications were ported as static PIE apps for Unikraft.
```
Comment (RaduNichita) [inline content/en/blog/2022-10-08-unikraft-hacktoberfest.md:25]:
```suggestion
While beginnings can sometimes be challenging and unrewarding, this was not the case for our contestants. After a few hours, the results were on the table: 3 applications were ported as static PIE apps for Unikraft.

File: content/en/blog/2022-10-08-unikraft-hacktoberfest.md [hunk 12]
```diff
@@ -5,23 +5,26 @@ author = "Radu Nichita"
...
+A great start! But wait, there's more! The participants continued working on their porting and submited their PRs. To our surprise we even had some PRs submited the next day.
```
Comment (RaduNichita) [inline content/en/blog/2022-10-08-unikraft-hacktoberfest.md:26]:
```suggestion
A great start! But wait, there's more! The participants continued working on their porting and submitted their PRs. To our surprise we even had some PRs submitted even the next day!.
```

**Review by RaduNichita — COMMENTED** [review]
Nice nitpicks, @mandrei12! Thanks for that. I left some minor comments and suggestions.

**Comment by razvand** [comment]
@RaduNichita , you can ignore the Linter issue, it's something to do with the image inclusion.

Please integrate the comments you did for @mandrei12 's contributions and use `Resolve conversation`. Squash commits into a single one. Then ask for a final review and, once done, we'll publish it.

File: content/en/blog/2022-10-08-unikraft-hacktoberfest.md [hunk 13]
```diff
@@ -0,0 +1,39 @@
++++
+title = "Unikraft Hacktoberfest '22"
+date = "2022-10-08T09:00:00+01:00"
+author = "Radu Nichita"
```
Comment (razvand) [inline content/en/blog/2022-10-08-unikraft-hacktoberfest.md:4]:
```suggestion
author = "Radu Nichita, Andrei Mutu"
```

File: content/en/blog/2022-10-08-unikraft-hacktoberfest.md [hunk 14]
```diff
@@ -0,0 +1,39 @@
...
+tags = ["hacktoberfest"]
++++
+
+### And Another One
+
+As tradition dictates, if it's Saturday then it is definitely another hackathon we are proud to host at Unikraft. Even so, the last one was a bit special. How you ask? Well, as we previously announced, we have been officially accepted to be part of the [Hacktoberfest 2022](https://hacktoberfest.com/).
```
Comment (razvand) [inline content/en/blog/2022-10-08-unikraft-hacktoberfest.md:10]:
```suggestion
As tradition dictates, if it's Saturday then it is definitely another hackathon we are proud to host at Unikraft.
Even so, the last one was a bit special.
How you ask?
Well, as we previously announced, we have been officially accepted to be part of the [Hacktoberfest 2022](https://hacktoberfest.com/).
```

**Review by razvand — CHANGES_REQUESTED** [review]
I like it. See my comments. Use one sentence per each line.

Then it will be OK.

File: content/en/blog/2022-10-08-unikraft-hacktoberfest.md [hunk 15]
```diff
@@ -0,0 +1,50 @@
...
+Well, as we previously announced, we have been officially accepted to be part of the [Hacktoberfest 2022](https://hacktoberfest.com/).
+
+Thus `UNIKRAFT HACKTOBERFEST '22` was born in the shape of a hybrid event.
+Here both familiar faces and newcomers alike came together physically at University POLITEHNICA of Bucharest or joined remotely on [the Unikraft Discord server](http://bit.ly/UnikraftDiscord) in order to test their knowledge to unikernels.
+
+### First Steps
+
+In order to break the ice, our documentation community lead, [Stefan Jumărea](https://github.com/StefanJum), gave an introductory presentation.
+This aimed to present both unikernels and Unikraft to the participants, introducing it to the newcomers and refreshing it for the veterans.
+Afterward, a short discussion about the challenges raised our participant's spirits.
```
Comment (mandrei12) [inline content/en/blog/2022-10-08-unikraft-hacktoberfest.md:22]: I think here is a typo. It should be `participants` without the apostrophe.

**Review by mandrei12 — CHANGES_REQUESTED** [review]
Besides the typo it looks good.

**Review by mandrei12 — APPROVED** [review]
It looks like everything is fine now.

**Review by razvand — APPROVED** [review]
All is good. Thanks, @RaduNichita , @mandrei12 

Reviewed-by: Razvan Deaconescu <razvan.deaconescu@cs.pub.ro>
Approved-by: Razvan Deaconescu <razvan.deaconescu@cs.pub.ro>

**Pull request merged**